main_view.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style_main_view.css">
+</head>
+<body>
+
+    
+<header>
+<div class="taskbar"> <span class="time"> 9:41 </span>  
+<img  class="status_phone" src="images/Cellular_Connection.svg" alt="Connection">  
+<img   class="status_phone" src="images/Wifiwifi_signal.svg" alt="Wifi_signal">
+<img class="status_phone" src="images/Batterybattery.svg" alt="Battery">
+</div>
+<div class="menu"> <span>Welcome</span> <img src="images/Enabledmenu_bar.svg" alt=""></div>
+</header>
+<main class="background_main">
+
+
+
+
+    <h1 class="introduction"> Hey there. I’m Jose Zepeda
+         <span>I’m a software developer</span> </h1>
+<p class="content_introduction">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
```

Comment:
- [ ] Kindly, Avoid lines of code that are too long (100 characters or more). Kindly fix this 👍🏻